Tools/Cluster.py

```diff
@@ -150,9 +125,8 @@ def Cluster_all(packdata, auxil, varlist, KK, logfile):
     # 4. Check if the selected sites are representative? (not sure)
 
     # 5. Output the ID
-    IDx = pd.concat([adict["PFT%itrainingID" % ii] for ii in kpfts])
-    IDx.drop_duplicates(subset=["lat", "lon"])
-    IDx = IDx.values
+    IDx = np.concatenate([adict["PFT%itrainingID" % ii] for ii in kpfts])
+    IDx = np.unique(IDx, axis=0)
```

Comment:
I haven'I haven't tested, but I assume these functions are equivalent.

Reply:
Yes, the changes here are because the data type of `IDx` has been changed from `pd.DataFrame` to `np.ndarray`.